ui/src/main/res/layout/activity_banktransfer_list.xml

```diff
@@ -0,0 +1,30 @@
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please see `CreditCardDetailsActivity` for the new layout format.

It uses `CoordinatorLayout` and also add a `RecyclerView` to show item details when amount is clicked.